src/lib/plannerUtils.ts

```diff
@@ -361,7 +361,6 @@ export function clonePlanner(sourcePlanner: PlannerData): PlannerData {
       courses: q.courses.map((crs) => {
         return lookup[crs].id;
```

Comment:
In this function, adjust how labels are cloned such that the label texts transfer when duplicating a planner